### ISSUE TITLE
Security vulnerability in dependency serialize-javascript 

### ISSUE THREAD

**Issue by sambulance** [humanmade/webpack-helpers#40]
Github is flagging that the `serialize-javascript` package contains an XSS vulnerability. 

npm audit:

```
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Moderate      │ Cross-Site Scripting                                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ serialize-javascript                                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=2.1.1                                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ @humanmade/webpack-helpers [dev]                             │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ @humanmade/webpack-helpers > copy-webpack-plugin >           │
│               │ serialize-javascript                                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/1426                            │
└───────────────┴──────────────────────────────────────────────────────────────┘
```

**Comment by kadamwhite** [comment]
Fixed in #43 , thanks @kevinlangleyjr and sorry @sambulance that I missed this issue!

**Comment by paulgibbs** [comment]
Thanks for fixing!
